presto-docs/src/main/sphinx/prestissimo/prestissimo-features.rst

```diff
@@ -184,3 +184,184 @@ for cleanup. Only applicable when ``enable-old-task-cleanup`` is ``true``.
 Old task is defined as a PrestoTask which has not received heartbeat for at least
 ``old-task-cleanup-ms``, or is not running and has an end time more than
 ``old-task-cleanup-ms`` ago.
+
+
+Session Properties
+------------------
+
+Defines all system session properties supported by native worker to ensure
```

Comment:
```suggestion
```
Suggest deleting this first sentence - it doesn't add meaning for the user that the topic heading Session Properties does not provide, and the "all" in "Defines all system session properties" is a strong promise to the user that we may not want to make.